Podcast/InternalProfileViewController.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import UIKit
+import Crashlytics
```

Comment:
is there a reason Crashlytics needs to be imported in this file? Doesn't look like we use it